### ISSUE TITLE
XML Error on Firefox

### ISSUE THREAD

**Issue by jsmueller7** [recogito/recogito-client#218]
(Tested with beta.5)

I have just given an introduction to Recogito. One of four participants could not open the TEI document. I know that it occurred with a Firefox browser. I have just successfully reproduced the problem myself. The window that's usually shows the document contains this error message:

```
XML-Verarbeitungsfehler: XML-Deklaration nicht wohlgeformt Adresse: https://<my-host>/de/annotate/636cda59-f5e9-4b09-81b4-95bd0813027f/0cc3ac16-da14-4a0d-9d78-10a828665367 Zeile Nr. 1, Spalte 16:<?xml version="1.1" encoding="UTF-8" standalone="no" ?><?xml-model href="textgrid:3q81v.0" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><?xml-model href="textgrid:3q81v.0" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="textgrid:3qgwb.0" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?> ---------------^
```

**Comment by rsimon** [comment]
Hi @jsmueller7, do you have the original TEI file? It looks like the file is actually invalid, so there may be little we can do (inside Recogito). But it would be useful to find out what exactly in the markup is breaking the rendering.

**Comment by jsmueller7** [comment]
> Hi @jsmueller7, do you have the original TEI file? It looks like the file is actually invalid, so there may be little we can do (inside Recogito). But it would be useful to find out what exactly in the markup is breaking the rendering.

Sure, sent you an email.

**Comment by rsimon** [comment]
Ok, this is a surprise to me. But it seems that Firefox does indeed not support XML v1.1. As I suspected above, this is nothing specific to Recogito. If you just try to open the file with Firefox directly, Firefox will reject it with the same error message. (Chrome will handle it fine, as you wrote.)

Because Recogito renders the XML markup directly, and relies on the browser's built-in XML engine to display it, there's nothing we can do in this particular case.

That being said: I suspect that the file is like not even using any of the XML 1.1-specific features. I simply edited the markup, and changed the first line from 

```xml
<?xml version="1.1" encoding="UTF-8" standalone="no" ?>
```

to 

```xml
<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

The file seemed to render fine then. I'd be hesitant to add such an override hack as a built-in feature in Recogito. But, as a workaround, the user could make this change outside of Recogito before uploading. The only other answer, I guess, would simply be that XML 1.1 documents are only supported on Chrome, but not FF.
